### DISCUSSION TITLE
is it possible de use the extension in command line ?

### DISCUSSION THREAD

**Discussion by webdevproformation** [facelessuser/MarkdownPreview#169] [Q&A] [answered]
Hello,

First what a great extension : i use it to create web site 
is it possible to use the extension out of submit text ? directly in shell command ?? 

Second point, when i use the fonction Save to html > there is 3 options : github / gitlab and markdown, is it possible to create his own version of html 

Thank a lot for the answer 

have a good day !

**Comment by facelessuser** [comment, marked as answer]
> is it possible to use the extension out of submit text ? directly in shell command ??

Currently, Markdown Preview is only designed as a Sublime Text plugin.

> Second point, when i use the fonction Save to html > there is 3 options : github / gitlab and markdown, is it possible to create his own version of html

I'm not quite sure what is being asked here, can you provide an example?

**Comment by webdevproformation** [comment]
First thank you for the answser.

To be more precise, in this screen : is it possible to add a new item in the dropdown menu 

![Capture d’écran 2023-08-29 111835](https://github.com/facelessuser/MarkdownPreview/assets/25311422/6fde0076-4a3b-43c0-8a25-4134ef9882ba)

**Comment by webdevproformation** [comment]
for instance a new item "only html version" or "html with it's own version of css " 

thx a lot

**Comment by facelessuser** [comment]
We document all the configuration that is possible in the docs. If you want to add a new parser.

We already offer a save to HTML option.

You can also add custom CSS.

I would check out the documentation and see what is possible and if you have specific questions, feel free to ask. With that said, it may not be able to be configured "exactly" how you imagine, but it is fairly flexible.
